### DISCUSSION TITLE
Using tablesaw in Java with ant.

### DISCUSSION THREAD

**Discussion by apostolos-geyer** [jtablesaw/tablesaw#948] [Q&A] [answered]
Hello,

I am relatively new to Java, at least when it comes to build tools. I have always used Java with Ant in Netbeans, as per the expectations of my Grade 11/12 Computer Science teacher, so I have never used Maven. That being the case, even in programming I do "for fun" I have used libraries by adding .jar's to my libraries folder. I want to use Tablesaw in a program, but I can't even figure out how to make a simple program in Maven, and I don't think I want to switch over right now either way. Is there any way to use this in Java with Ant? 

Sorry if this is a stupid question lol

**Comment by lwhite1** [comment, marked as answer]
Hi @pauljamesgeyer,

No it doesn't work with Ant.  You have to use Maven. 

Maven is easier to use than Ant; it's just different. Basically, you need to install Maven. Then create a project where you'll do your work, putting a "pom" file in the root directory. Then run maven in that directory. I attached a simple example pom with Tablesaw as the only dependency.  (Github doesn't let you attach xml files so it's zipped)

If you setup your project correctly, you can probably build and run from Netbeans, which no doubt, has maven support built in. Otherwise, you can use maven at the command line, running "mvn clean" or "mvn clean install"

Ant is rarely used in Java anymore. Once you get used to Maven it will seem easier.  

[pom.xml.zip](https://github.com/jtablesaw/tablesaw/files/6546663/pom.xml.zip)

**Reply by apostolos-geyer** [reply]
Thanks, I made a test project in it with Maven yesterday and managed to import [this COVID data](https://data.ontario.ca/dataset/confirmed-positive-cases-of-covid-19-in-ontario/resource/455fd63b-603d-4608-8216-7d8647f43350) and take a look at a few different things. Thanks!

**Comment by lwhite1** [comment]
Excellent.  Have fun.
